### PR TITLE
travis setup

### PR DESCRIPTION
**Pull request by leena-jawale**

### CODE REVIEW

File: pkg/apis/cluster/v1alpha1/clustercurator_types.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+	// +required
+	Action Action `json:"action,omitempty"`
+
+	// PreJobs holds the job specifications
+	// +optional
+	PreJobs []JobSpec `json:"prejobs"`
+
+	// PostJobs holds the job specifications
+	// +optional
+	PostJobs []JobSpec `json:"postjobs"`
```
Comment (itdove) [inline pkg/apis/cluster/v1alpha1/clustercurator_types.go:39]: We didn't foreseen to have an array but that's fine.

File: pkg/apis/cluster/v1alpha1/clustercurator_types.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+	// Jobs holds the job specifications
+	// +optional
+	Jobs []JobConfig `json:"jobs"`
```
Comment (itdove) [inline pkg/apis/cluster/v1alpha1/clustercurator_types.go:43]: Should be JobSpec.... I guess in the yaml the Job was missing the RunOnAction

Comment (leena-jawale) [reply pkg/apis/cluster/v1alpha1/clustercurator_types.go:43]: it is missing type and RunOnAction. so I removed those. But I guess we can use same `JobSpec` here.

Comment (itdove) [reply pkg/apis/cluster/v1alpha1/clustercurator_types.go:43]: yes JobConfig and JobSpec are the same and so we can drop JobSpec

**Review by Randy424 — APPROVED** [review]

**Review by Randy424 — COMMENTED** [review]
Thanks for doing this, it looks great. I wanted to throw a review just to communicate that I went over the PR, but beyond Dominique's clarification/comments I have nothing to add.

**Pull request merged**